utilities/new_client.py

```diff
@@ -0,0 +1,297 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage: python new_client.py -v <api version> -s <service name> -c <path to config>
+
+import argparse
+import copy
+import os
+import re
+import sys
+import subprocess
+from jinja2 import Environment, FileSystemLoader
```

Comment:
can you put a new line between standard library imports and third-party imports?